lighthouse-core/test/report/html/renderer/details-renderer-test.js

```diff
@@ -385,7 +385,7 @@ describe('DetailsRenderer', () => {
       assert.equal(urlEl.localName, 'div');
       assert.equal(urlEl.title, urlText);
       assert.equal(urlEl.dataset.url, urlText);
-      assert.ok(urlEl.firstChild.classList.contains('lh-text'));
+      assert.equal(urlEl.firstChild.nodeName, 'A');
```

Comment:
can you expand the tests of this anchor (here and below) now that it's more than just an `.lh-text` with the url? Can probably c/p from the `renders link values` test (though that doesn't appear to check the `.lh-text__url-host` stuff?)